plugin/secrets_access_token.go

```diff
@@ -80,18 +82,29 @@ func (b *backend) secretAccessTokenRenew(ctx context.Context, req *logical.Reque
 	return logical.ErrorResponse("short-term access tokens cannot be renewed - request new access token instead"), nil
 }
 
+func isInvalidTokenErr(err error) bool {
+	gErr, ok := err.(*googleapi.Error)
+	return ok && gErr.Code == 400 && strings.Contains(gErr.Body, "invalid_token")
```

Comment:
Quick check -- this would be a 400 not a 403? 403 is likely more proper but that doesn't mean it's what Google's using :-)

Reply:
Yup! This server will only return 400 or 200 and nothing else! So great!!